requirements_dev.txt

```diff
@@ -8,7 +8,7 @@ tabulate
 numpy>=1.9.1
 scipy>=0.9.0
 blessings>=1.5.1
-pandas>=0.15.2
+pandas==0.18.1
```

Comment:
Are we fixing pandas? Or we should say >=0.18.1 (or actually latest version is 0.21.?)